elastic/kibana
# [ResponseOps] Reduce the number of times we call the field_caps API for the ES Query rule

## Summary

We are not able to reduce the field_cap API response for the ES Query rule for now, but while testing I noticed that we were calling the field_cap API twice. This PR removes one of those calls that is unnecessary.


### Checklist




### To verify

- Add a log statement at the start of the `getFieldCapabilities` function in `src/plugins/data_views/server/fetcher/lib/field_capabilities/field_capabilities.ts`
- Start kibana
- Create a KQL ES Query rule
- Verify that when the rule is executed `getFieldCapabilities` is only called once by verifying that the log statement only appears once.
